Adding note for automatically track deep links (#4338)
segmentio/segment-docs: src/connections/sources/catalog/libraries/mobile/ios/index.md

{% endcodeexampletab %}
{% endcodeexample %}

> Please note that you'll still need to call the `continueUserActivity` and `openURL` methods on the analytics client.

Reviewer: ```suggestion
> note ""
> **Note:** You still need to call the `continueUserActivity` and `openURL` methods on the analytics client.
```
